### PR TITLE
evaluation function script + test

### PR DESCRIPTION
**Pull request by adrianl726**
And it runs finally :)

### CODE REVIEW

File: scripts/model_evaluation_wine_predictor.py [hunk 1]
```diff
@@ -38,6 +39,12 @@ def main(train_data, test_data, pipeline_path, table_to, plot_to, seed):
     np.random.seed(seed)
     set_config(transform_output="pandas")
 
+    # Create paths to save tables and plots if they do not exist
```
Comment (Farhan-Faisal) [inline scripts/model_evaluation_wine_predictor.py:42]: nice

File: tests/test_evaluation.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# test_evaluation.py
```
Comment (Farhan-Faisal) [inline tests/test_evaluation.py:1]: temp_path would have been better. But if it works, it works^

**Review by Farhan-Faisal — APPROVED** [review]
awesome! we r almost done!

File: tests/test_evaluation.py (hunk 2 above)
Comment (adrianl726) [reply tests/test_evaluation.py:1]: I agree but it doesn't work for me at first so I resorted to this.

**Pull request merged**